pkg/scheduling/nodetemplate.go

```diff
@@ -27,22 +27,26 @@ import (
 // the fields in Provisioner. These structs are maintained separately in order
 // for fields like Requirements to be able to be stored more efficiently.
 type NodeTemplate struct {
+	ProvisionerName      string
 	Provider             *v1alpha5.Provider
 	Labels               map[string]string
 	Taints               Taints
 	StartupTaints        Taints
 	Requirements         Requirements
 	KubeletConfiguration *v1alpha5.KubeletConfiguration
+	Limits               *v1alpha5.Limits
```

Comment:
It feels wrong to me to include this in the node template. We're converging this object towards `provisioner`. When I designed this struct, I was careful to limit it to the parameters inherent to the node, and removed any that relate to the orchestration of that node (e.g. limits, ttls, knowledge of provisioner).